### PR TITLE
Change README to md format

### PR DESCRIPTION
**Pull request by haoxli**
- Change README to md format
- Update suite.json
- Update README.md format

BUG=https://crosswalk-project.org/jira/browse/XWALK-5274

### CODE REVIEW

**Comment by jennycao** [comment]
@Honry Pls help to review the PR too about WRT part readme change.

**Comment by Honry** [comment]
LGTM

File: tools/xmlsimplifier/README.md [hunk 1]
```diff
@@ -1,32 +1,41 @@
...
+
   -f, --file         Specify an test definition file (tests.xml)
   -o, --output       Specify an output file
   -p, --platform     Specify a platform
   -e, --execution    Specify a execution type
 
-HOW TO:
-    1.Pre-condition
+## HOW TO Config:
+
```
Comment (zqzhang) [inline tools/xmlsimplifier/README.md]: => How to Config

File: tools/xmlsimplifier/README.md [hunk 2]
```diff
@@ -41,13 +51,17 @@ HOW TO:
         element_nodes = description,pre_condition,steps,step,step_desc,expected,test_script_entry
 
 
-    3.HOW to USE
+## HOW to USE
```
Comment (zqzhang) [inline tools/xmlsimplifier/README.md]: How to Use

**Comment by jennycao** [comment]
@haoxli Pls also adjust the format of following files
crosswalk-test-suite/embeddingapi/embedding-asyncapi-android-tests/README.md 
crosswalk-test-suite/misc/sampleapp-android-tests/README.md 
crosswalk-test-suite/usecase/usecase-apptools-tests/README.md 
crosswalk-test-suite/usecase/usecase-wrt-auto-tests/README.md

**Comment by zqzhang** [comment]
Merge as-is for further development.

**Pull request merged**